### PR TITLE
Add OME Community Meeting program

### PR DESCRIPTION
**Pull request by lunson**
- Adds Jason's draft schedule from https://docs.google.com/spreadsheets/d/125S_W1xovW8r7GAKxCX8e_0KwzTMqU7UqZh1Ty644k4/edit#gid=0, replaces "???" with "TBD"; 
- I'll change the layout for videos once we actually have videos
- Adds a landing page, which will serve as placeholder for the registration details along with  Zoom/conference instructions when we finalize those details
- Adds a Speakers page for us to add photos/bios

### CODE REVIEW

File: events/ome-community-meeting-2020/speakers/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - firstname: Petr
+      surname: Walczysko
+      job_title: QA Software Sepcialist
+      bio: "Petr Walczysko joined the OME project in October 2012 as a software specialist for testing and quality assurance. He studied at Charles University of Prague where he received Master of Science degree in Physics and the University of Freiburg in Germany where he received PhD in Biology. Throughout his PhD studies and his further career as a researcher he was intensively using conventional, confocal and multiphoton fluorescence microscopy applications on biological systems. He was adapting these optical microscopy techniques for particular biological problems, and also worked on the subsequent image analysis of microscopic images in a range of image analysis programmes. He enjoys yoga, reading and chess."
```
Comment (mtbc) [inline events/ome-community-meeting-2020/speakers/index.html:29]:
```suggestion
      bio: "Petr Walczysko joined the OME project in October 2012 as a software specialist for testing and quality assurance. He studied at Charles University of Prague where he received Master of Science degree in Physics and the University of Freiburg in Germany where he received PhD in Biology. Throughout his PhD studies and his further career as a researcher he was intensively using conventional, confocal and multiphoton fluorescence microscopy applications on biological systems. He was adapting these optical microscopy techniques for particular biological problems, and also worked on the subsequent image analysis of microscopic images in a range of image analysis programs. He enjoys yoga, reading and chess."
```
Might want to get this one too. :smiley:

**Comment by mtbc** [comment]
I'm wondering if the lunch / coffee on Day 3 is intended as we don't have such on the previous.

File: events/ome-community-meeting-2020/speakers/index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      website_name: "Glencoe Software"
+      website_link: "glencoesoftware.com"
+      twitter: "jrswedlow"
+    - firstname: Sebastien
+      surname: Besson
+      job_title: Senior Software Developer
+      bio: "Sebastien Besson joined the OME team as a developer in march 2012. Originally trained as a physicist, he received his PhD from the Unversite Pierre at Marie Curie in Paris. He then carried out postdoctoral research at the interface between physics and biology at Harvard University. In 2011, Sebastien joined the Danuser lab to convert in-house image analysis into turn-key software packages for the cell biology community. He became part of the main Dundee team in 2015."
```
Comment (mtbc) [inline events/ome-community-meeting-2020/speakers/index.html:17]:
```suggestion
      bio: "Sebastien Besson joined the OME team as a developer in March 2012. Originally trained as a physicist, he received his PhD from the Unversite Pierre at Marie Curie in Paris. He then carried out postdoctoral research at the interface between physics and biology at Harvard University. In 2011, Sebastien joined the Danuser lab to convert in-house image analysis into turn-key software packages for the cell biology community. He became part of the main Dundee team in 2015."
```

File: events/ome-community-meeting-2020/speakers/index.html (hunk 1 above)
Comment (lunson) [reply events/ome-community-meeting-2020/speakers/index.html]: Thanks, Mark!

File: events/ome-community-meeting-2020/speakers/index.html (hunk 2 above)
Comment (lunson) [reply events/ome-community-meeting-2020/speakers/index.html]: Good eye! :)

**Comment by lunson** [comment]
@mtbc And yes, the lunch/coffee blurb is intentional. I mostly just wanted to break the monotony of bullet point/"details to follow soon" pattern since the 3rd Day doesn't have any "Relevant Video Talks" blocks. I'm totally open to better suggestions!

File: events/ome-community-meeting-2020/speakers/index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    - firstname: Josh
+      surname: Moore
+      job_title: Senior Software Architect
+      bio: "Josh Moore is an American developer living in Walluf, Germany, with his wife and two kids. With a background in machine learning and distributed computing, Josh began work on the OMERO Java server in the Spring of 2005 as part of the Swedlow Lab, after evaluating use of the OME Perl server for the Mitocheck project while in the iBios group at the DKFZ, Heidelberg, Germany. Before that he was a German-language pre-school teacher at a Montessori school in Alabama, of all things. He's a lisp-loving, Aikido-practicing, vegan with interests in RDF, HDF, and various other three letter acronyms."
+    - firstname: Petr
+      surname: Walczysko
+      job_title: QA Software Sepcialist
```
Comment (pwalczysko) [inline events/ome-community-meeting-2020/speakers/index.html:28]:
```suggestion
      job_title: QA Software Specialist
```

**Comment by pwalczysko** [comment]
@lunson @jrswedlow I think here is a misunderstanding regarding the workshops (Day 3).
My hunch is that @lunson (understandably) interpreted the workshops with the stuff in the right-hand side of the [gdoc sheet](https://docs.google.com/spreadsheets/d/125S_W1xovW8r7GAKxCX8e_0KwzTMqU7UqZh1Ty644k4/edit#gid=0), which says things like ``OMERO analysis workflows`` and ``OME Ontology...``.
I would be puzzled by workshops which take only 25 minutes each. Also a workshop on OME Ontology is highly improbable. Instead, I think that the Day 3 is actually not scheduled in detail at all, all we know about day 3 atm is that there will be some workshops. The things in cells J42 and higher of the [gdoc sheet](https://docs.google.com/spreadsheets/d/125S_W1xovW8r7GAKxCX8e_0KwzTMqU7UqZh1Ty644k4/edit#gid=0) are simply a remnant of the older version, which was not deleted (and as I remember the old version, these cells pertained to Day 2). Is that so @jrswedlow ?

**Comment by lunson** [comment]
Thanks @pwalczysko - Yup, I mistakenly thought Jason had cleaned up the worksheet after you mentioned it at the team meeting. I have edits from him that I'll be working on and will push today.

File: events/ome-community-meeting-2020/day2/index.html [hunk 4]
```diff
@@ -29,15 +29,15 @@
...
+      vid_speaker1: "Jean-Marie Burel, OME Dundee"
       vid_title2: Napari
-      vid_speaker2: Nicholas Soforniew
+      vid_speaker2: Nicholas Soforniew, Chan Zuckerberg Initiative
       vid_title3: QuPath
-      vid_speaker3: "Pete Bankhead"
+      vid_speaker3: "Pete Bankhead, University of Edinburgh"
       vid_title4: scOMERO
-      vid_speaker4: "Ola Tarkowska & Omer Bayraktar"
+      vid_speaker4: "Ola Tarkowska & Omer Bayraktar, wellcome Sanger Centre"
```
Comment (pwalczysko) [inline events/ome-community-meeting-2020/day2/index.html:38]: wellcome with lower case ``w`` sounds wrong - you sure about this ?

File: events/ome-community-meeting-2020/workshops/index.html [hunk 5]
```diff
@@ -3,43 +3,19 @@
...
+<div class="row">
+    <div class="small-12 medium-6 medium-offset-3 columns">
+        <div class="card card-support">
+            <div class="card-section">
+                <i class="border-green icon-green fa fa-comments-o fa-2x"></i>
+                <h4>Workshops TBD</h4>
+                <p class="card-caption">
+                    Workshops are still to be determined but if you are interested in running one, please reach out to <a href="mailto:J.Matthew@dundee.ac.uk?&cc=j.r.swedlow@dundee.ac.uk?&subject=I want to run a workshop!">June Matthew and Jason Swedlow</a>.
+                </p>
+                <p class="text-center"><a href="mailto:J.Matthew@dundee.ac.uk?&cc=j.r.swedlow@dundee.ac.uk?&subject=I want to run a workshop!" class="btn-blue button small">I want to run a workshop!</a></p>
```
Comment (pwalczysko) [inline events/ome-community-meeting-2020/workshops/index.html:17]: This nice trick with mail to June and cc jason does not work on some setups. For example, my default email for the OS is not the one I really use. When I try to copy the email address by right-clicking on it (in order to paste it into my real email app), only June's address gets copied and Jason's is completely lost for me - I would suggest not putting Jason as a cc, but instead as the second addressee would fix the issue.

**Comment by pwalczysko** [comment]
@lunson thanks, Day 3 now seems to me all right. Just two minor questions/comments. The rest of the commits seem fine to me too. Let @jrswedlow judge.
Screen Shot 2020-04-20 at 15 37 42

**Comment by lunson** [comment]
Jason approved to go live in https://trello.com/c/RIcmFL52/21-virtual-agenda. I'll put any additional edits from him in another PR. Thanks.

File: events/ome-community-meeting-2020/day1/index.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+      details: Details to follow soon.
+    - type: Special Topic
+      title: "Project/Institutional Resources"
+      videos: yes
+      vid_title1: Pancreatlas
+      vid_speaker1: "Marcela Brissova, Vanderbilt"
+      vid_title2: AICS
+      vid_speaker2: Graham Johnson, Allen Institute of Cell Science
+      vid_title3: HTAN
+      vid_speaker3: "Peter Sorger, Harcard Medical School"
```
Comment (manics) [inline events/ome-community-meeting-2020/day1/index.html:36]:
```suggestion
      vid_speaker3: "Peter Sorger, Harvard Medical School"
```

**Comment by pwalczysko** [comment]
Checked all the programme on https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ome-community-meeting-2020/ - all the changes look good, build looks good, new icons to attract attention to topics look well, latest batch of commits all implemented successfully.

As @jrswedlow is happy as well, merging and deploying.

**Comment by lunson** [comment]
Thanks, guys!

**Pull request merged**